dashboard/test/controllers/lti_v1_controller_test.rb

```diff
@@ -515,6 +516,28 @@ def create_valid_jwt_raise_error
     assert_equal @integration.lti_deployments.count, 1
   end
 
+  test 'auth - link lti_user_identity with lti_deployment' do
+    payload = get_valid_payload
+    jwt = create_jwt_and_stub(payload)
+    user = create_preexisting_user(payload)
+    deployment = LtiDeployment.create(deployment_id: @deployment_id, lti_integration_id: @integration.id)
+    assert_equal deployment.lti_user_identities.count, 0
+    post '/lti/v1/authenticate', params: {id_token: jwt, state: @state}
+    assert_equal deployment.lti_user_identities.count, 1
+    assert_equal deployment.lti_user_identities.first, user.lti_user_identities.first
+  end
+
+  test 'auth - do not link lti_user_identity with lti_deployment if already linked' do
+    payload = get_valid_payload
+    jwt = create_jwt_and_stub(payload)
+    user = create_preexisting_user(payload)
+    deployment = LtiDeployment.create(deployment_id: @deployment_id, lti_integration_id: @integration.id)
```

Comment:
nit: could change deployment to an instance variable `@deployment` and reuse it between tests. Could also use the factory instead of LtiDeployment.create